### DISCUSSION TITLE
Zróżnicowanie czasu wysyłki dla różnych rynków Allegro

### DISCUSSION THREAD

**Discussion by DanielCyran** [allegro/allegro-api#7752] [Suggestions - rejected]
Witam,

Proszę o zróżnicowanie czasu wysyłki na produktach dla różnych rynków Allegro. Obecnie, przy aktualizacji produktów poprzez API, czas wysyłki na Allegro Czechy jest automatycznie ustawiany na taki sam jak na Allegro Polska.

JEST TO PILNA FUNKCJA

**Comment by Lukasz-Zurek** [comment]
@DanielCyran Temat ten konsultuję jeszcze z odpowiednim zespołem. Wkrótce wrócę do Ciebie z informacją.

**Comment by Lukasz-Zurek** [comment]
@DanielCyran Potwierdzam, że dostępne do wyboru deklaracje są takie same. Jedna oferta może być na wielu serwisach (obecnie mam tu na myśli dodatkowy allegro-cz) i na wszystkich ma taką samą deklaracje wysyłki.

Jeżeli Sprzedający nadaje przesyłki allegro-pl i allegro-cz to analizujemy wszystkie Jego wysyłki. Paczki niezależnie od docelowego kraju są nadawane w ten sam sposób i odbierane przez tych samych przewoźników. Dlatego nie zakładamy specjalnego rozróżnienia w przewidywaniu czasu nadania.

Rozróżnienie jest oczywiście w czasach przewozu, ale one są niezależne od Sprzedającego.

Nie pozostaje mi nic innego jak poprosić Cię o doprecyzowanie - na czym konkretnie zależałoby Tobie w kontekście zróżnicowania czasu wysyłki, jaka przemawia za tym potrzeba. Propozycję możesz w całości opisać w dedykowanym do tego miejscu, czyli w zakładce [Discussions](https://github.com/allegro/allegro-api/discussions).

**Comment by DanielCyran** [comment]
@Lukasz-Zurek 
Witam,
W naszej firmie aktualnie procesy są zorganizowane w taki sposób, że potrzebujemy mieć możliwość wydłużenia czasu wysyłki paczek do Czech o jeden lub dwa dni.

Nie wszystkie magazyny obsługują wysyłki międzynarodowe i nie pozwalają na wysyłki międzynarodowe. Stąd konieczność wewnętrznych przesunięć międzymaganyzownych co powoduje, że ciężko w wielu przypadkach zaoferować czas wysyłki w takim samym czasie jak wysyłki do klientów z Polski. I to dotyczy każdych krajów, nie tylko Czech.

Obecnie istnieje możliwość ustawienia jednego czasu dostawy dla dwóch rynków. Możemy więc ustawić czas dostawy na trzy / cztery dni, ale nie chcemy zwiększać czasu wysyłki dla Polski o jeden lub dwa dni.

W związku z tym prosimy o możliwość oddzielenia czasu dostawy dla Polski i dla Czech.

**Reply by Lukasz-Zurek** [reply]
@DanielCyran Przeniosłem Twój wątek do [Discussions](https://github.com/allegro/allegro-api/discussions). Zmiana, o której piszesz, wiązałaby się z rozdzieleniem "czasu wysyłki" i możliwością definiowania go wg. serwisu, np. allegro-pl i allegro-cz. Dziękuję za szczegółowy opis i przesłaną sugestię.

**Comment by DanielCyran** [comment]
@Lukasz-Zurek  czy wiadomo już coś ??  wprowadzicie to ?

**Reply by Lukasz-Zurek** [reply]
Nie. Dodam tylko, że taka zmiana wiązałaby się z rozróżnieniem definiowania czasu wysyłki osobno dla allegro-pl, allegro-cz, itd. Na ten moment nie mamy tego w planach, pozostawiam jednak ten wątek w celu późniejszej analizy.

**Comment by DanielCyran** [comment]
@Lukasz-Zurek W sumie......... czego ja oczekiwałem zgłaszając wam poważną potrzebę, jeśli wy nie potraficie sobie poradzić z innym, prostszym problemem, jakim jest zidentyfikowanie nie działającego przycisku "Zapisz" w moich aplikacjach? W skrócie, działacie po prostu słabo.

**Reply by Lukasz-Zurek** [reply]
@DanielCyran Dziękujemy za opinię. Jeżeli chodzi o wszelkiego rodzaju pomysły dotyczące usprawnienia API, szczególnie te, które zmieniają pewną logikę - a takiego pomysłu dotyczy Twój wątek - zakładam, że znasz skalę serwisu Allegro i tu proszę Cię o wyrozumiałość, ponieważ wdrażanie zmian w API jest złożonym procesem, zależnym również od zespołów odpowiadających za dane usługi. Jeżeli ewentualna zmiana dotyka kilku usług - sprawa się komplikuje. Pominę fakt, iż jesteś jedyną osobą, która taką sugestię przesłała, a nie ukrywajmy - z Allegro API korzystają również inni użytkownicy.

Jeżeli natomiast chodzi o problem z przyciskiem "Zapisz" - przekazaliśmy już wcześniej ten temat do odpowiedniego zespołu. Proszę o cierpliwość.

**Comment by DanielCyran** [comment]
@Lukasz-Zurek 
Rozumiem, że skala Allegro i złożoność wprowadzania zmian sprawiają, że nowy rynek Allegro Czechy jest trudnym tematem. Natomiast nie każdy sprzedawca zgłasza problemy, jeśli je napotyka. Jednakże, jako że jest to problem logiczny, powinien zostać rozwiązany jak najszybciej, nawet nie ze względu na sprzedawców, a dla dobra klientów końcowych. Poprawne informacje o czasie wysyłki są ważne, aby klienci byli zadowoleni i nie dokonywali zwrotów, co przekłada się również na zaufanie do platformy Allegro.    

Brak możliwości zróżnicowania czasu wysyłki dla różnych rynków Allegro wpłynie negatywnie na:

**Zadowolenie z zakupów przez klientów końcowych:**
Klienci końcowi będą częściej dostawać błędne informacje o czasie wysyłki.
Klienci końcowi będą rozczarowani, gdy ich zamówienie nie dotrze w oczekiwanym czasie.
Klienci końcowi mogą dokonać zwrotu, co może prowadzić do negatywnych opinii na temat sprzedawcy i zwiększonych kosztów.

**Zaufanie do zakupów dokonywanych na Allegro Czechy:**
Klienci końcowi mogą być zniechęceni do dokonywania zakupów na Allegro Czechy, jeśli będą mieli poczucie, że otrzymują nieprawdziwe informacje.
Zaufanie do platformy Allegro może zostać podkopane, jeśli klienci końcowi będą mieć problemy z otrzymaniem zamówienia w oczekiwanym czasie.

**Liczba negatywów na kontach sprzedawców:**
Klienci końcowi, którzy otrzymają zamówienie z opóźnieniem, mogą złożyć negatywną opinię na temat sprzedawcy.
Liczba negatywów na kontach sprzedawców może wzrosnąć, co może prowadzić do utraty zaufania klientów i zmniejszenia sprzedaży.

**Reply by Lukasz-Zurek** [reply]
Dziękuję. Przekażę te informacje osobie odpowiedzialnej za czas dostawy i wysyłki w Allegro.

**Comment by DanielCyran** [comment]
Witam. 
@Lukasz-Zurek  @MartaNowaczyk @PawelTaberski @MaciejPotepa @PatrykBocian 
Proszę o konkretną informację, ponieważ do dzisiaj nie została wprowadzona opcja umożliwiająca różnicowanie czasu wysyłki w zależności od rynku. Wkrótce wprowadzany będzie kolejny rynek. Proszę, aby Allegro umożliwiło sprzedawcom udostępnianie rzetelnych informacji dotyczących czasu wysyłki. Jest to kwestia priorytetowa, ponieważ klient końcowy musi otrzymywać prawdziwe informacje, a sprzedawca powinien być w stanie je dostarczyć. Pozdrawiam i liczę na wprowadzenie tej oczywistej zmiany.

**Reply by Lukasz-Zurek** [reply]
@DanielCyran Konsultowaliśmy już ten temat z osobą odpowiedzialną za czasy wysyłki i dostawy i jako Serwis nie przewidujemy jakichkolwiek zmian w tym zakresie. Podkreślę jednak, że nie jest to decyzja zależna od naszego zespołu wsparcia API. Jeśli się coś zmieni to poinformujemy o tym fakcie w tym wątku. Póki co obecny wątek pozostawimy, by również pozostali użytkownicy mogli oddać swój głos na Twój pomysł.

**Comment by szopi99** [comment]
W moim odczuciu faktycznie przydałoby się takie rozróżnienie w zakresie czasu wysyłki pomiędzy rynkami polskim, a czeskim. Konto Allegro: SZOPiSHOP. Mam sporo produktów na rynku czeskim i stanowiłoby to spore ułatwienie.

**Comment by DanielCyran** [comment]
@Lukasz-Zurek 
Ilu sprzedawców musi potwierdzić konieczność wprowadzenia rozróżnienia czasu wysyłki dla każdego rynku, abyście umożliwili sprzedawcom podawanie prawdziwych informacji klientowi końcowemu związanego z czasem wysyłki?

**Reply by Lukasz-Zurek** [reply]
To nie jest pytanie do mnie czy też mojego zespołu. Po naszej stronie zrobiliśmy wszystko, by naświetlić ten temat właścicielom usługi. Tego typu zmiana wiązałaby się z modyfikacją na szerszą skalę, w kontekście całego Allegro, w związku z czym wszelkie dodatkowe uwagi kieruj proszę korzystając z [formularza kontaktowego](https://allegro.pl/pomoc/kontakt?kategoria=713f3db7-e741-4fba-9404-09ee86a58e6c).

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy wprowadzenia omawianych tutaj zmian.
